src/styles/_catalog.scss

```diff
@@ -1,22 +1,26 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+  grid-template-columns: repeat(auto-fill, $width);
 
   max-width: 944px;
```

Comment:
Don't think you need to use max-width here in this task

Reply:
U don't need to hard-code :(

But, U can use variables and calculate the value with the formula